src/mudlet.pro

```diff
@@ -108,8 +108,17 @@ $$system(git --version, blob, GIT_STATUS)
 isEqual(GIT_STATUS, 0) {
     # Git command appeared to work - so get a description of the current
     # repository state
-    GIT_DESCRIPTION=$$system(git describe --all --dirty --broken)
-    GIT_COMMITHASH=$$system(git rev-parse --short HEAD)
+    isEmpty($$(GITHUB_PR_BRANCH)) {
```

Comment:
As I said - I am not sure that using the environmental variables directly works as expected. I would suggest you do this - also I think there is a logic error in the second `if`as you are trying to do something *with* a value that you have **just tested for it to be empty**:
```
    GITHUB_PR_BRANCH_TEST=$$(GITHUB_PR_BRANCH)
    isEmpty(GITHUB_PR_BRANCH_TEST) {
        GIT_DESCRIPTION=$$system(git describe --all --dirty --broken)
    } else {
        GIT_DESCRIPTION=$${GITHUB_PR_BRANCH_TEST}
    }
    GITHUB_SHA_TEST=$$(GITHUB_SHA)
    isEmpty(GITHUB_SHA_TEST) {
        GIT_COMMIT="HEAD"
    } else {
        GIT_COMMIT=$${GITHUB_SHA_TEST}^2

    }
```

Reply:
So complicated, so many workarounds for something simple like an environment variable :wink:

Reply:
I forgot to push my changes.